client/app/services/parameters/TextParameter.js

```diff
@@ -15,6 +15,13 @@ class TextParameter extends Parameter {
     }
     return normalizedValue;
   }
+
+  getExecutionValue() {
+    if (!trim(this.value)) {
+      return null;
+    }
+    return this.value;
+  }
```

Comment:
Before:
<img width="317" alt="Screen Shot 2019-11-01 at 9 05 58" src="https://user-images.githubusercontent.com/486954/68008233-e340c200-fc86-11e9-9ab6-4a3056e816c3.png">


After:
<img width="317" alt="Screen Shot 2019-11-01 at 9 06 10" src="https://user-images.githubusercontent.com/486954/68008234-e340c200-fc86-11e9-9c9a-d1f5a76eb109.png">

Reply:
I didn't see a reason why someone would use a parameter only containing whitespaces, so this looks good to me :+1:.